go.mod

```diff
@@ -5,11 +5,10 @@ go 1.13
 require (
```

Comment:
you should change to go "1,14" on line above

Reply:
Done.